src/main/java/RacingCar/RacingCar.java

```diff
@@ -0,0 +1,27 @@
+package RacingCar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCar {
+
+    private final List<Car> carList = new ArrayList<>();
+    public int tryCnt;
```

Comment:
```suggestion
    private final int tryCnt;
```
이렇게 선언할 수 있으면 불변함이 증명되어 부수효과가 발생하지 않도록 할 수 있어요.
이렇게 할려면 생성자를 사용해봐야 할거 같네요😀